benchmarks/src/garage_benchmarks/experiments/baselines/gaussian_cnn_baseline.py

```diff
@@ -10,8 +10,7 @@
 from garage.tf.policies import CategoricalCNNPolicy
 
 params = {
-    'conv_filters': (32, 64, 64),
-    'conv_filter_sizes': (5, 3, 2),
+    'conv_filters': ((32, (5, 5)), (64, (3, 3)), (64, (2, 2))),
```

Comment:
it would be nice to format these as below in all examples, to demonstrate to users better what the argument means

```
(
    (32, (5, 5)), 
    (64, (3, 3)), 
    (64, (2, 2)),
)  # yapf: disable
```